juriscraper/opinions/united_states_backscrapers/state/massappct.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+from juriscraper.opinions.united_states_backscrapers.state import mass
+
+
+class Site(mass.Site):
+    backscrape_date_range_mapper = [
+        {
+            "start": datetime(2021, 7, 9),
+            "end": None,
+            "url": "http://masscases.com/app100-124.html",
+        },
+        {
+            "start": datetime(2009, 8, 20),
+            "end": datetime(2021, 5, 12),
+            "url": "http://masscases.com/app75-99.html",
+        },
```

Comment:
are these the only ones we need backscraped?  any reason we cant make this more dynamic?